core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpScreen.kt

```diff
@@ -0,0 +1,152 @@
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+import androidx.navigation.NavController
+import org.kiwix.kiwixmobile.core.R
+import org.kiwix.kiwixmobile.core.error.DiagnosticReportActivity
+import org.kiwix.kiwixmobile.core.extensions.ActivityExtensions.start
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun HelpScreen(
+  modifier: Modifier = Modifier,
+  data: List<HelpScreenItemDataClass>,
+  navController: NavController
+) {
+  val context = LocalContext.current
+
+  val isDarkTheme = isSystemInDarkTheme()
+
+  val backgroundColor =
+    if (isDarkTheme) colorResource(id = R.color.mine_shaft_gray900) else Color.White
+  val dividerColor =
+    if (isDarkTheme) colorResource(id = R.color.mine_shaft_gray600) else colorResource(
+      id = R.color.mine_shaft_gray350
+    )
+
+  Scaffold(
+    modifier = Modifier.fillMaxSize(),
+    topBar = {
+      TopAppBar(
+        title = {
+          Text(
+            modifier = modifier.padding(start = 16.dp),
+            text = stringResource(id = R.string.menu_help),
+            color = Color.White // Set title text color to white
+          )
+        },
+        navigationIcon = {
+          IconButton(onClick = navController::popBackStack) {
+            Icon(
+              imageVector = Icons.Filled.ArrowBack,
+              contentDescription = "Back",
+              tint = Color.White // Set navigation icon color to white
+            )
+          }
+        },
+        colors = TopAppBarDefaults.topAppBarColors(
+          containerColor = Color.Black // Set top app bar background color to black
+        )
+      )
+    },
+    containerColor = backgroundColor
+  ) {
+
+    Column(
+      modifier = Modifier
+        .padding(it)
+    ) {
+      Row(
+        modifier = Modifier
+          .fillMaxWidth()
+          .clickable {
+            (context as? Activity)?.start<DiagnosticReportActivity>()
+          },
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Start
+      ) {
+        Image(
+          painter = painterResource(R.drawable.ic_feedback_orange_24dp),
+          contentDescription = "Feedback",
```

Comment:
The `contentDescription` should be `Send diagnostic report` as it is for "Send diagnostic report" we have a string for this `send_report` which we are using in the `fragment_help.xml` so we should use that.